spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/task/TaskExecutionProperties.java

```diff
@@ -36,6 +39,22 @@
 	 */
 	private String threadNamePrefix = "task-";
 
+	/**
+	 * The maximum number of time that the executor is supposed to block on shutdown in
```

Comment:
Configuration properties descriptions do not start with "The", "A", etc. The first sentence must be much shorter as it's used by IDEs on auto-completion.

Reply:
Wasn't aware about that. Will change it to

>The maximum number of time that the executor is supposed to block on shutdown waiting for remaining tasks to complete.

Is this text acceptable?

Reply:
No it is isn't. It starts with `The` and I've indicated previously we're not documenting keys that way.

On a second thought those two keys are related to shutdown so it might be interesting to crete a sub group for them the same way we did for pool

Reply:
Sorry, I actually removed `The` in my commit. Yes I'll move them into a shutdown sub group